// @nuxthub/db/schema.mjs
import { integer, real, sqliteTable, text } from "drizzle-orm/sqlite-core";
import { sql } from "drizzle-orm";
import { bigint, boolean, integer as integer$1, jsonb, pgTable, real as real$1, serial, text as text$1, timestamp, uniqueIndex, varchar } from "drizzle-orm/pg-core";

//#region rolldown:runtime
var __defProp = Object.defineProperty;
var __exportAll = (all, symbols) => {
	let target = {};
	for (var name in all) {
		__defProp(target, name, {
			get: all[name],
			enumerable: true
		});
	}
	if (symbols) {
		__defProp(target, Symbol.toStringTag, { value: "Module" });
	}
	return target;
};

//#endregion
//#region server/db/schema.sqlite.ts
var schema_sqlite_exports = /* @__PURE__ */ __exportAll({
	admins: () => admins$2,
	apiKeys: () => apiKeys$2,
	cards: () => cards$2,
	failures: () => failures$2,
	logs: () => logs$2,
	oauthAccounts: () => oauthAccounts$2,
	orders: () => orders$2,
	paymentMethods: () => paymentMethods$2,
	posts: () => posts$2,
	products: () => products$2,
	settings: () => settings$2,
	subscriptions: () => subscriptions$2,
	themeSettings: () => themeSettings$2,
	users: () => users$2,
	visitorEvents: () => visitorEvents$2,
	visitorProfiles: () => visitorProfiles$2,
	webhooks: () => webhooks$2
});
const users$2 = sqliteTable("users", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	email: text("email").notNull().unique(),
	passwordHash: text("password_hash"),
	nickname: text("nickname"),
	avatarUrl: text("avatar_url"),
	lastLoginAt: integer("last_login_at", { mode: "timestamp" }),
	CashBalance: integer("cash_balance", { mode: "number" }).default(0),
	GrantBalance: integer("grant_balance", { mode: "number" }).default(0),
	GrantExpiresAt: integer("grant_expires_at", { mode: "timestamp" }),
	TierLevel: integer("tier_level", { mode: "number" }).default(0),
	status: integer("status").default(1),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const admins$2 = sqliteTable("admins", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	username: text("username").notNull().unique(),
	passwordHash: text("password_hash").notNull(),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const oauthAccounts$2 = sqliteTable("oauth_accounts", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	userId: integer("user_id").notNull().references(() => users$2.id),
	provider: text("provider").notNull(),
	providerAccountId: text("provider_account_id").notNull(),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const products$2 = sqliteTable("products", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	slug: text("slug").unique(),
	name: text("name").notNull(),
	price: real("price").notNull(),
	description: text("description"),
	content: text("content"),
	type: text("type").notNull(),
	imageUrl: text("image_url"),
	views: integer("views").notNull().default(0),
	imageUrls: text("image_urls", { mode: "json" }).$type(),
	resource: text("resource"),
	isActive: integer("is_active", { mode: "boolean" }).notNull().default(true),
	metaData: text("meta_data", { mode: "json" }),
	sortOrder: integer("sort_order").notNull().default(0),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const cards$2 = sqliteTable("cards", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	productId: integer("product_id").notNull().references(() => products$2.id),
	cardNumber: text("card_number").notNull(),
	isUsed: integer("is_used", { mode: "boolean" }).notNull().default(false),
	orderId: text("order_id"),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const paymentMethods$2 = sqliteTable("payment_methods", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	name: text("name").notNull(),
	code: text("code").notNull(),
	iconUrl: text("icon_url"),
	isActive: integer("is_active", { mode: "boolean" }).notNull().default(false),
	configJson: text("config_json"),
	info: text("info"),
	create: text("create"),
	callback: text("callback"),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const orders$2 = sqliteTable("orders", {
	id: text("id").primaryKey(),
	amount: real("amount").notNull(),
	productId: integer("product_id").notNull().references(() => products$2.id),
	userId: integer("user_id").references(() => users$2.id),
	contactEmail: text("contact_email").notNull(),
	payMethod: text("pay_method"),
	tradeNo: text("trade_no"),
	status: text("status").notNull().default("none"),
	deliveryInfo: text("delivery_info"),
	metaData: text("meta_data", { mode: "json" }),
	visitorId: text("visitor_id"),
	subscriptionId: text("subscription_id"),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`),
	paidAt: integer("paid_at", { mode: "timestamp" }),
	payStatus: text("pay_status").notNull().default("pending")
});
const subscriptions$2 = sqliteTable("subscriptions", {
	id: text("id").primaryKey(),
	gatewaySubId: text("gateway_sub_id"),
	userId: integer("user_id").references(() => users$2.id),
	productId: integer("product_id").notNull().references(() => products$2.id),
	payMethod: text("pay_method").notNull(),
	status: text("status").notNull().default("active"),
	interval: text("interval").notNull(),
	intervalCount: integer("interval_count").notNull().default(1),
	amount: real("amount").notNull(),
	currency: text("currency").notNull().default("USD"),
	currentPeriodStart: integer("current_period_start", { mode: "timestamp" }),
	currentPeriodEnd: integer("current_period_end", { mode: "timestamp" }),
	cancelAtPeriodEnd: integer("cancel_at_period_end", { mode: "boolean" }).default(false),
	metaData: text("meta_data", { mode: "json" }),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`),
	updatedAt: integer("updated_at", { mode: "timestamp" }).default(sql`CURRENT_TIMESTAMP`)
});
const settings$2 = sqliteTable("settings", {
	key: text("key").primaryKey(),
	value: text("value").notNull(),
	description: text("description"),
	updatedAt: integer("updated_at", { mode: "timestamp" }).default(sql`(unixepoch() * 1000)`)
});
const themeSettings$2 = sqliteTable("theme_settings", {
	themeName: text("theme_name").primaryKey(),
	config: text("config").notNull(),
	updatedAt: integer("updated_at", { mode: "timestamp" }).default(sql`(unixepoch() * 1000)`)
});
const apiKeys$2 = sqliteTable("api_keys", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	name: text("name", { length: 50 }),
	keyString: text("key_string", { length: 64 }).notNull().unique(),
	userId: integer("user_id").references(() => users$2.id),
	status: integer("status").default(1),
	orderId: text("order_id").references(() => orders$2.id),
	productId: integer("product_id").references(() => products$2.id),
	quotaLimit: integer("quota_limit", { mode: "number" }).notNull().default(0),
	quotaUsed: integer("quota_used", { mode: "number" }).notNull().default(0),
	allowedModels: text("allowed_models", { mode: "json" }).$type(),
	isActive: integer("is_active", { mode: "boolean" }).notNull().default(true),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const failures$2 = sqliteTable("failures", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	orderId: text("order_id").notNull(),
	cardBin: text("card_bin"),
	reason: text("reason").notNull(),
	amount: real("amount"),
	payMethod: text("pay_method"),
	contactEmail: text("contact_email"),
	rawResponse: text("raw_response"),
	visitorId: text("visitor_id"),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const webhooks$2 = sqliteTable("webhooks", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	name: text("name").notNull(),
	url: text("url").notNull(),
	events: text("events", { mode: "json" }).$type(),
	secret: text("secret"),
	isActive: integer("is_active", { mode: "boolean" }).notNull().default(true),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const logs$2 = sqliteTable("logs", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	level: text("level").notNull().default("info"),
	message: text("message").notNull(),
	details: text("details"),
	source: text("source"),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const visitorProfiles$2 = sqliteTable("visitor_profiles", {
	visitorId: text("visitor_id").primaryKey(),
	userId: integer("user_id").references(() => users$2.id),
	firstSeenAt: integer("first_seen_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`),
	lastSeenAt: integer("last_seen_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`),
	landingPath: text("landing_path"),
	firstPath: text("first_path"),
	lastPath: text("last_path"),
	firstReferrer: text("first_referrer"),
	lastReferrer: text("last_referrer"),
	firstSourceType: text("first_source_type"),
	lastSourceType: text("last_source_type"),
	firstSource: text("first_source"),
	lastSource: text("last_source"),
	firstMedium: text("first_medium"),
	lastMedium: text("last_medium"),
	firstCampaign: text("first_campaign"),
	lastCampaign: text("last_campaign"),
	firstContent: text("first_content"),
	lastContent: text("last_content"),
	firstTerm: text("first_term"),
	lastTerm: text("last_term"),
	country: text("country"),
	region: text("region"),
	city: text("city"),
	locale: text("locale"),
	currency: text("currency"),
	deviceType: text("device_type"),
	browser: text("browser"),
	os: text("os"),
	userAgent: text("user_agent"),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`),
	updatedAt: integer("updated_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const visitorEvents$2 = sqliteTable("visitor_events", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	visitorId: text("visitor_id").notNull(),
	userId: integer("user_id").references(() => users$2.id),
	orderId: text("order_id").references(() => orders$2.id),
	productId: integer("product_id").references(() => products$2.id),
	eventName: text("event_name").notNull(),
	eventAction: text("event_action"),
	path: text("path"),
	referrer: text("referrer"),
	sourceType: text("source_type"),
	source: text("source"),
	medium: text("medium"),
	campaign: text("campaign"),
	content: text("content"),
	term: text("term"),
	country: text("country"),
	region: text("region"),
	city: text("city"),
	locale: text("locale"),
	currency: text("currency"),
	deviceType: text("device_type"),
	browser: text("browser"),
	os: text("os"),
	userAgent: text("user_agent"),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`)
});
const posts$2 = sqliteTable("posts", {
	id: integer("id").primaryKey({ autoIncrement: true }),
	key: text("key"),
	sort: integer("sort"),
	slug: text("slug").notNull().unique(),
	title: text("title").notNull(),
	description: text("description"),
	content: text("content"),
	type: text("type").notNull().default("blog"),
	imageUrl: text("image_url"),
	views: integer("views").notNull().default(0),
	isActive: integer("is_active", { mode: "boolean" }).notNull().default(true),
	metaData: text("meta_data", { mode: "json" }),
	createdAt: integer("created_at", { mode: "timestamp" }).notNull().default(sql`CURRENT_TIMESTAMP`),
	updatedAt: integer("updated_at", { mode: "timestamp" }).default(sql`CURRENT_TIMESTAMP`)
});

//#endregion
//#region server/db/schema.pg.ts
var schema_pg_exports = /* @__PURE__ */ __exportAll({
	admins: () => admins$1,
	apiKeys: () => apiKeys$1,
	cards: () => cards$1,
	failures: () => failures$1,
	logs: () => logs$1,
	oauthAccounts: () => oauthAccounts$1,
	orders: () => orders$1,
	paymentMethods: () => paymentMethods$1,
	posts: () => posts$1,
	products: () => products$1,
	settings: () => settings$1,
	subscriptions: () => subscriptions$1,
	themeSettings: () => themeSettings$1,
	users: () => users$1,
	visitorEvents: () => visitorEvents$1,
	visitorProfiles: () => visitorProfiles$1,
	webhooks: () => webhooks$1
});
const users$1 = pgTable("users", {
	id: serial("id").primaryKey(),
	email: text$1("email").notNull().unique(),
	passwordHash: text$1("password_hash"),
	nickname: text$1("nickname"),
	avatarUrl: text$1("avatar_url"),
	lastLoginAt: timestamp("last_login_at", { withTimezone: true }),
	CashBalance: bigint("cash_balance", { mode: "bigint" }).default(sql`0`),
	GrantBalance: bigint("grant_balance", { mode: "bigint" }).default(sql`0`),
	TierLevel: integer$1("tier_level").default(0),
	GrantExpiresAt: timestamp("grant_expires_at", { withTimezone: true }),
	status: integer$1("status").default(1),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const admins$1 = pgTable("admins", {
	id: serial("id").primaryKey(),
	username: text$1("username").notNull().unique(),
	passwordHash: text$1("password_hash").notNull(),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const oauthAccounts$1 = pgTable("oauth_accounts", {
	id: serial("id").primaryKey(),
	userId: integer$1("user_id").notNull().references(() => users$1.id),
	provider: text$1("provider").notNull(),
	providerAccountId: text$1("provider_account_id").notNull(),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
}, (table) => {
	return { providerIdx: uniqueIndex("provider_account_idx").on(table.provider, table.providerAccountId) };
});
const products$1 = pgTable("products", {
	id: serial("id").primaryKey(),
	slug: text$1("slug").unique(),
	name: text$1("name").notNull(),
	price: real$1("price").notNull(),
	description: text$1("description"),
	content: text$1("content"),
	type: text$1("type").notNull(),
	imageUrl: text$1("image_url"),
	views: integer$1("views").notNull().default(0),
	imageUrls: jsonb("image_urls").$type(),
	resource: text$1("resource"),
	isActive: boolean("is_active").notNull().default(true),
	metaData: jsonb("meta_data"),
	sortOrder: integer$1("sort_order").notNull().default(0),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const cards$1 = pgTable("cards", {
	id: serial("id").primaryKey(),
	productId: integer$1("product_id").notNull().references(() => products$1.id),
	cardNumber: text$1("card_number").notNull(),
	isUsed: boolean("is_used").notNull().default(false),
	orderId: text$1("order_id"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const paymentMethods$1 = pgTable("payment_methods", {
	id: serial("id").primaryKey(),
	name: text$1("name").notNull(),
	code: text$1("code").notNull(),
	iconUrl: text$1("icon_url"),
	isActive: boolean("is_active").notNull().default(false),
	configJson: text$1("config_json"),
	info: text$1("info"),
	create: text$1("create"),
	callback: text$1("callback"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const orders$1 = pgTable("orders", {
	id: text$1("id").primaryKey(),
	amount: real$1("amount").notNull(),
	productId: integer$1("product_id").notNull().references(() => products$1.id),
	userId: integer$1("user_id").references(() => users$1.id),
	contactEmail: text$1("contact_email").notNull(),
	payMethod: text$1("pay_method"),
	tradeNo: text$1("trade_no"),
	status: text$1("status").notNull().default("none"),
	deliveryInfo: text$1("delivery_info"),
	metaData: jsonb("meta_data"),
	visitorId: text$1("visitor_id"),
	subscriptionId: text$1("subscription_id"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow(),
	paidAt: timestamp("paid_at", { withTimezone: true }),
	payStatus: text$1("pay_status").notNull().default("pending")
});
const subscriptions$1 = pgTable("subscriptions", {
	id: text$1("id").primaryKey(),
	gatewaySubId: text$1("gateway_sub_id"),
	userId: integer$1("user_id").references(() => users$1.id),
	productId: integer$1("product_id").notNull().references(() => products$1.id),
	payMethod: text$1("pay_method").notNull(),
	status: text$1("status").notNull().default("active"),
	interval: text$1("interval").notNull(),
	intervalCount: integer$1("interval_count").notNull().default(1),
	amount: real$1("amount").notNull(),
	currency: text$1("currency").notNull().default("USD"),
	currentPeriodStart: timestamp("current_period_start", { withTimezone: true }),
	currentPeriodEnd: timestamp("current_period_end", { withTimezone: true }),
	cancelAtPeriodEnd: boolean("cancel_at_period_end").default(false),
	metaData: jsonb("meta_data"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow(),
	updatedAt: timestamp("updated_at", { withTimezone: true }).defaultNow()
});
const settings$1 = pgTable("settings", {
	key: text$1("key").primaryKey(),
	value: text$1("value").notNull(),
	description: text$1("description"),
	updatedAt: timestamp("updated_at", { withTimezone: true }).defaultNow()
});
const themeSettings$1 = pgTable("theme_settings", {
	themeName: text$1("theme_name").primaryKey(),
	config: text$1("config").notNull(),
	updatedAt: timestamp("updated_at", { withTimezone: true }).defaultNow()
});
const apiKeys$1 = pgTable("api_keys", {
	id: serial("id").primaryKey(),
	name: varchar("name", { length: 50 }).notNull(),
	keyString: varchar("key_string", { length: 64 }).notNull().unique(),
	userId: integer$1("user_id").references(() => users$1.id),
	status: integer$1("status").default(1),
	tierLevel: integer$1("tier_level").notNull().default(0),
	orderId: text$1("order_id").references(() => orders$1.id),
	productId: integer$1("product_id").references(() => products$1.id),
	quotaLimit: bigint("quota_limit", { mode: "number" }).notNull().default(0),
	quotaUsed: bigint("quota_used", { mode: "number" }).notNull().default(0),
	allowedModels: jsonb("allowed_models").$type(),
	isActive: boolean("is_active").notNull().default(true),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const failures$1 = pgTable("failures", {
	id: serial("id").primaryKey(),
	orderId: text$1("order_id").notNull(),
	cardBin: text$1("card_bin"),
	reason: text$1("reason").notNull(),
	amount: real$1("amount"),
	payMethod: text$1("pay_method"),
	contactEmail: text$1("contact_email"),
	rawResponse: text$1("raw_response"),
	visitorId: text$1("visitor_id"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const webhooks$1 = pgTable("webhooks", {
	id: serial("id").primaryKey(),
	name: text$1("name").notNull(),
	url: text$1("url").notNull(),
	events: jsonb("events").$type(),
	secret: text$1("secret"),
	isActive: boolean("is_active").notNull().default(true),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const logs$1 = pgTable("logs", {
	id: serial("id").primaryKey(),
	level: text$1("level").notNull().default("info"),
	message: text$1("message").notNull(),
	details: text$1("details"),
	source: text$1("source"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const visitorProfiles$1 = pgTable("visitor_profiles", {
	visitorId: text$1("visitor_id").primaryKey(),
	userId: integer$1("user_id").references(() => users$1.id),
	firstSeenAt: timestamp("first_seen_at", { withTimezone: true }).notNull().defaultNow(),
	lastSeenAt: timestamp("last_seen_at", { withTimezone: true }).notNull().defaultNow(),
	landingPath: text$1("landing_path"),
	firstPath: text$1("first_path"),
	lastPath: text$1("last_path"),
	firstReferrer: text$1("first_referrer"),
	lastReferrer: text$1("last_referrer"),
	firstSourceType: text$1("first_source_type"),
	lastSourceType: text$1("last_source_type"),
	firstSource: text$1("first_source"),
	lastSource: text$1("last_source"),
	firstMedium: text$1("first_medium"),
	lastMedium: text$1("last_medium"),
	firstCampaign: text$1("first_campaign"),
	lastCampaign: text$1("last_campaign"),
	firstContent: text$1("first_content"),
	lastContent: text$1("last_content"),
	firstTerm: text$1("first_term"),
	lastTerm: text$1("last_term"),
	country: text$1("country"),
	region: text$1("region"),
	city: text$1("city"),
	locale: text$1("locale"),
	currency: text$1("currency"),
	deviceType: text$1("device_type"),
	browser: text$1("browser"),
	os: text$1("os"),
	userAgent: text$1("user_agent"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow(),
	updatedAt: timestamp("updated_at", { withTimezone: true }).notNull().defaultNow()
});
const visitorEvents$1 = pgTable("visitor_events", {
	id: serial("id").primaryKey(),
	visitorId: text$1("visitor_id").notNull(),
	userId: integer$1("user_id").references(() => users$1.id),
	orderId: text$1("order_id").references(() => orders$1.id),
	productId: integer$1("product_id").references(() => products$1.id),
	eventName: text$1("event_name").notNull(),
	eventAction: text$1("event_action"),
	path: text$1("path"),
	referrer: text$1("referrer"),
	sourceType: text$1("source_type"),
	source: text$1("source"),
	medium: text$1("medium"),
	campaign: text$1("campaign"),
	content: text$1("content"),
	term: text$1("term"),
	country: text$1("country"),
	region: text$1("region"),
	city: text$1("city"),
	locale: text$1("locale"),
	currency: text$1("currency"),
	deviceType: text$1("device_type"),
	browser: text$1("browser"),
	os: text$1("os"),
	userAgent: text$1("user_agent"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow()
});
const posts$1 = pgTable("posts", {
	id: serial("id").primaryKey(),
	key: text$1("key"),
	sort: integer$1("sort"),
	slug: text$1("slug").notNull().unique(),
	title: text$1("title").notNull(),
	description: text$1("description"),
	content: text$1("content"),
	type: text$1("type").notNull().default("blog"),
	imageUrl: text$1("image_url"),
	views: integer$1("views").notNull().default(0),
	isActive: boolean("is_active").notNull().default(true),
	metaData: jsonb("meta_data"),
	createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow(),
	updatedAt: timestamp("updated_at", { withTimezone: true }).defaultNow()
});

//#endregion
//#region server/db/schema.ts
const resolveDialect = () => {
	const explicitDialect = process.env.DB_DIALECT?.replace(/"/g, "").toLowerCase();
	if (explicitDialect === "postgresql" || explicitDialect === "sqlite") return explicitDialect;
	const connectionUrl = process.env.DATABASE_URL || process.env.POSTGRES_URL || process.env.POSTGRESQL_URL || process.env.NUXT_DATABASE_URL || process.env.LIBSQL_URL || "";
	if (connectionUrl.startsWith("postgres://") || connectionUrl.startsWith("postgresql://")) return "postgresql";
	return "sqlite";
};
const activeSchema = resolveDialect() === "postgresql" ? schema_pg_exports : schema_sqlite_exports;
const { users, admins, oauthAccounts, products, cards, paymentMethods, orders, settings, themeSettings, apiKeys, failures, webhooks, logs, visitorProfiles, visitorEvents, posts, subscriptions } = activeSchema;

//#endregion
